docs/osbuild-composer.7.rst

```diff
@@ -0,0 +1,77 @@
+================
+osbuild-composer
+================
+
+------------------------
+OSBuild Composer Service
+------------------------
+
+:Manual section: 7
+:Manual group: Miscellaneous
+
+DESCRIPTION
+===========
+
+The composer project is a set of HTTP services for composing operating system
+images. It builds on the pipeline execution engine of *osbuild* [#osbuild]_ and
+defines its own class of images that it supports building.
+
+Multiple APIs are available to access a composer service. This includes
+support for the *lorax-composer* [#lorax-github]_ API, and as
+such can serve as drop-in replacement for lorax-composer.
+
+You can control a composer instance either directly via the provided APIs, or
+through higher-level user-interfaces from external projects. This, for
+instance, includes a *Cockpit Module* [#cockpit-composer]_ or using the
+*composer-cli* [#composer-cli]_ command-line tool.
+
+Frontends
+---------
+
+*Composer* does not ship with frontends itself. However, several external
+frontends for *Composer* already exist. These include:
+
+**Cockpit Composer**
+    This module for *Cockpit* [#cockpit]_ allows a great level of control of a
+    *Composer* instance running on a cockpit-managed machine.
```

Comment:
Do you mind calling this "Image Builder module for Cockpit" or something similar? We're not referring to it as "Composer" in the UI.

Technically, "Cockpit" is only the project name and we always use "Fedora Web Console" in documentation. It's probably fine to keep using "Cockpit" here though, because it's intended for an audience that knows about these things.

Reply:
Yeah, makes sense.

I am still a bit unsure how to phrase things in different contexts. This needs more experience and trial-and-error I guess.

Regarding Cockpit, do you have a reference to the documentation you talk about? They do have their upstream-project-documentation (eg. [[1]](https://cockpit-project.org/guide/latest/feature-storaged.html)), but this consistently speaks of `Cockpit`. I guess you were referring to RHEL documentation (or at least more downstream docs)? I would like to get a better feel of how they pulled this off. I am still unsure how exactly we will achieve something similar.

[1] https://cockpit-project.org/guide/latest/feature-storaged.html

Reply:
I think a useful differentiation is upstream vs downstream documentation: upstream it's all "Cockpit" (like in the link you provided), downstream it's "X Web Console".

Not super important to get exactly right. I just wanted to make you aware of it.